scripts/restore-sanitised-database.sh

```diff
@@ -6,6 +6,8 @@ LOCAL_DB_PASSWORD="password"
 LOCAL_DB_USER="postgres"
 LOCAL_DB_NAME="data_store"
 DB_IMAGE="postgres:16.2"
+RESTORE_DB=${1}
```

Comment:
The reason this parameter is set in this way is due to the following:

- If someone wants to restore a sanitised database locally, they can use the Flask command db-data seed-sanitised-db. This will only restore the database to the local environment because the `seed_sanitised_db()`  function automatically passes the 'y' argument to the script, which is why the default is set to handle this case.
- 
- If a user wants to restore the sanitised database to an AWS (dev/test) environment, they would typically run the restore script from the terminal using ./scripts/restore-sanitised-database.sh. This way script will not have any arguments and will prompt the user to confirm if they want to restore the database locally. If they respond 'n' (No), the script will proceed to restore the database to the AWS environment based on the profile selected via AWS Vault. Before we were doing only this but flask command has been added so it can be restored locally from cli.


Yes I can change the name **RESTORE_DB** to more clear name like you said **RESTORE_TO_LOCAL_DB**.

Reply:
Thanks @israr-ulhaq, understood on the reasons but a named argument would be better even if that is only used in the context of the Flask CLI command, just for clarity

Sounds good re the variable name change 👍